analyzer/src/main/kotlin/managers/Yarn.kt

```diff
@@ -69,6 +70,9 @@ class Yarn(
 
     override fun getRemotePackageDetails(workingDir: File, packageName: String): JsonNode {
         val process = run(workingDir, "info", "--json", packageName)
-        return jsonMapper.readTree(process.stdout)["data"]
+        val yarnErr = jsonMapper.readTree(process.stderr)["data"]
+        return jsonMapper.readTree(process.stdout)["data"] ?: yarnErr.also {
```

Comment:
Doing it this way instead of how @mnonnenmacher proposed has the drawback that it's now always tried to deserialize from `stderr`, which might fail if there is no error.

Reply:
fixed

Reply:
But now you need drop the `yarnErr` variable 😉

Reply:
done